content/en/api/users/code_snippets/api-user-get-app-key.sh

```diff
@@ -0,0 +1,7 @@
+api_key="<DATADOG_API_KEY>"
+app_key="<DATADOG_APPLICATION_KEY>"
+user_handle="<USER_HANDLE>"
+
+curl --request GET "https://api.datadoghq.com/api/v2/users/${user_handle}/application_keys" \
```

Comment:
This route is actually not accessible with API/APP key authentication. It only works with a logged in user session. 

The other issue is that it would be that the route doesn't exist for v2, only for v1.

Reply:
Let's remove this endpoint for now then.

Reply:
Let's remove this endpoint for now then.